src/Framework/ClassResolver/GlobalInstance/AnonymousGlobal.php

```diff
@@ -20,6 +20,14 @@ final class AnonymousGlobal
     /** @var array<string,object> */
     private static array $cachedGlobalInstances = [];
 
+    /**
+     * @internal
+     */
+    public static function resetCache(): void
```

Comment:
The method is marked with `@ignore`, even though, I would add a comment explaining why this method is public and what to do if the client want to reset the cache, I mean, which other alternatives exists (if it makes sense) and why is discouraged to call this method directly from somewhere else outside Gacela.
I think there are also a few public methods with `@ignore` keyword, same for them, what do you think?

Reply:
I thought/think the function name is self explanatory enough 🤔
As you commented, there are multiple other similar functions for the other resetCache parts. 
<img width="993" alt="Screenshot 2023-03-18 at 13 44 09" src="https://user-images.githubusercontent.com/5256287/226106813-01b86cd4-13dc-4f8f-9262-11a1729deeb7.png">

---

If any dev would try to use an internal method, it's by their own responsibility. They will get a warning from psalm/phpstan telling them that the method is internal, so they should not use it.
<img width="908" alt="Screenshot 2023-03-18 at 13 47 32" src="https://user-images.githubusercontent.com/5256287/226106968-9a5e24d6-8b13-4d3e-a8f6-a77821c96a86.png">